.github/workflows/maven-release.yml

```diff
@@ -1,74 +1,68 @@
+    with:
+      tag: ${{ github.event.inputs.tag }}
+      create-github-release: ${{ github.event.inputs.create-github-release == 'true' }}
+      release-variant: ${{ github.event.inputs.extension }}
+      version-validation-name: ${{ github.event.inputs.extension }}
+      version-validation-paths: code/gradle.properties:properties_multi_module
+    secrets: inherit
   publish:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-
-      - name: Set up Java
-        uses: actions/setup-java@v3
-        with:
-          distribution: 'zulu'
-          java-version: 17
-
-      - name: Cache Gradle packages
-        uses: actions/cache@v2
-        with:
-          path: ~/.gradle/caches
-          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
-          restore-keys: ${{ runner.os }}-gradle
-
-      - name: Verify version        
-        run: |
-          set -eo pipefail
-          echo Release version: ${{ github.event.inputs.tag }}
-          (./scripts/version.sh -n ${{  github.event.inputs.extension }} -v ${{ github.event.inputs.tag }})
-      
-      - name: Create GH Release
-        id: create_release
-        uses: release-drafter/release-drafter@v5
-        if: ${{ github.event.inputs.action_tag == 'yes' }}
-        with:
-          name: v${{ github.event.inputs.tag }}-${{  github.event.inputs.extension }}
-          tag: v${{ github.event.inputs.tag }}-${{  github.event.inputs.extension }}
-          version: v${{ github.event.inputs.tag }}-${{  github.event.inputs.extension }}
-          publish: true
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  
-
-      - name: Import GPG key
-        env:
-          GPG_SECRET_KEYS: ${{ secrets.GPG_SECRET_KEYS }}
-          GPG_OWNERTRUST: ${{ secrets.GPG_OWNERTRUST }}
-        run: |
-          echo $GPG_SECRET_KEYS | base64 --decode | gpg --import --no-tty --batch --yes
-          echo $GPG_OWNERTRUST | base64 --decode | gpg --import-ownertrust --no-tty --batch --yes
-
-      - name: Publish to Maven Central Repository
-        run: make ${{  github.event.inputs.extension }}-publish-main
-        env:
-          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
-          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
-          GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
-          GPG_KEY_ID: ${{ secrets.GPG_KEY_ID }}
\ No newline at end of file
+    if: ${{ github.event.inputs.extension != 'core' }}
+    permissions:
+        contents: write
+    uses: adobe/aepsdk-commons/.github/workflows/android-maven-release.yml@gha-android-3.3.0
+    with:
+      tag: ${{ github.event.inputs.tag }}
+      create-github-release: ${{ github.event.inputs.create-github-release == 'true' }}
+      release-variant: ${{ github.event.inputs.extension }}
+      version-validation-name: ${{ github.event.inputs.extension }}
+      version-validation-paths: code/gradle.properties:properties_multi_module
```

Comment:
Can you also add validation for version in code files so that we don't miss it during the release?